### ISSUE TITLE
Serialization can't handle inf

### ISSUE THREAD

**Issue by radarsat1** [siconos/siconos#216]
This is a separate serialization bug from #213.
It appears that when _T=inf, boost throws an exception on reading it.
https://stackoverflow.com/questions/28696663/boost-serialization-1-5-5-crash-when-meets-nan-and-inf

In KernelTest::t5 this is due to the `_T` member of Simulation which is not initialized yet from the NSDS, but even if that were fixed, it wouldn't fix the larger issue of handling inf, since that is a valid value for finalT, so I make an issue here until we find a work-around.

**Comment by radarsat1** [comment]
This is only a problem for xml files I believe, it shouldn't affect binary archives.

By the way I just experimented with removing `_T` from Simulation, do we need it there?  It was only like a 3-or 4-line change.  In principle it can be different from NSDS->finalT(), but then we just set it in EventsManager directly from there and skip storing it in Simulation, seems to work well enough.

This doesn't fix it though.  I just realized I made a mistake in the issue description, it is not `Simulation::_T` that causes a problem, but `EventsManager::_T`, which maybe just needs to be initialized in the right place.  (In t5, a simulation step is not executed, so not all initialization takes place with the new scheme.)
